examples/plot_interpolate_bad_channels.py

```diff
@@ -51,7 +49,7 @@
 evoked_before.plot(exclude=[])
 
 # compute interpolation (also works with Raw and Epochs objects)
-evoked_after = evoked_before.copy().interpolate_bads_eeg()
+evoked_after = evoked_before.interpolate_bads()
 
-# plot interpolated (prevsious bads)
+# plot interpolated (previous bads)
 evoked_after.plot(exclude=[])
```

Comment:
I'm okay with that. We should warn clearly in the docstring, probably directly beneath the one-line description (as opposed to in the `Notes` section). We could also have a check in inverse modeling if any channel name ends with `_virtual` (or whatever we add) we give a stern warning. It wouldn't be bulletproof but it would help.